features/cart/shopping_cart/adding_product_with_discounted_catalog_price_to_cart.feature

```diff
@@ -0,0 +1,28 @@
+@shopping_cart
+Feature: Adding a simple product with discounted catalog price to the cart
+    In order to select products with proper price
+    As a Visitor
+    I want to be able to add simple products with discounted catalog price to the cart
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a "T-Shirt" configurable product
+        And this product has "PHP T-Shirt" variant priced at "$20.00"
+        And the store has a "Mug" configurable product
+        And this product has "PHP Mug" variant priced at "$5.00"
+        And there is a catalog promotion "Winter sale" that reduces price by "25%" and applies on "PHP T-Shirt" variant
+
+    @todo
+    Scenario: Adding a simple product with discounted catalog price to the cart
+        When I add this product to the cart
+        Then I should be on my cart summary page
+        And I should be notified that the product has been successfully added
+        And I should see "PHP T-Shirt" with unit price "$15.00" in my cart
+
+    @todo
+    Scenario: Adding a simple product with discounted catalog price to the cart as a logged in customer
```

Comment:
I'm wondering if this scenario is needed 🤔 It does not have another result than the previous one, so does not bring a real value checking it